### PR TITLE
Available services without tests.

### PR DESCRIPTION
**Pull request by youaremysky99**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddServiceCommand
+     * and returns an AddServiceCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddServiceCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_DESCRIPTION, PREFIX_COST, PREFIX_ID);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_DESCRIPTION, PREFIX_COST, PREFIX_ID)
```
Comment (hmtey) [inline src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java:28]: Code seems to require user to input `i/ID` but this prefix is used for person id.

also, `AddServiceCommand` states that service id is supposed to be automatically generated and not via user input?

**Review by hmtey — CHANGES_REQUESTED** [review]
Remember to update UG!

File: src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java (hunk 1 above)
Comment (youaremysky99) [reply src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java:28]: Ah, I rethink about it and see it's almost impossible to generate these Ids properly.

Comment (youaremysky99) [reply src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java:28]: also, it's just syntax and pattern recognizer. I think it's okay, just for parsing commands.

Comment (hmtey) [reply src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java:28]: Hmm then do we still charge the services by service id? (If not i think this command might be of not much use to the receptionist)

If we are I suggest asking for user input -> new prefix + update command usage message!

Comment (youaremysky99) [reply src/main/java/seedu/address/logic/parser/AddServiceCommandParser.java:28]: yes, but I mean /i is just used for parsing, not really affect the functioning. And the hotel will show a list of services with ID for the receptionist to refer to. If not, he/she will need to remember all the costs of all services.

**Pull request merged**